java/ql/src/experimental/Security/CWE/CWE-489/StrutsBad.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!DOCTYPE struts PUBLIC
+	"-//Apache Software Foundation//DTD Struts Configuration 2.3//EN"
+	"http://struts.apache.org/dtds/struts-2.3.dtd">
+
+<struts>
+    <constant name="struts.enable.DynamicMethodInvocation" value="true" />
+    <constant name="struts.devMode" value="true" />
+    <constant name="struts.i18n.encoding" value="utf-8"></constant>
```

Comment:
Not really important, but for consistency it might be good to use a self-closing tag here (as well as for `StrutsGood.xml` and for the test files).
```suggestion
    <constant name="struts.i18n.encoding" value="utf-8" />
```